sdk/tests/NodeAddressBookTest.cc

```diff
@@ -0,0 +1,40 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "NodeAddressBook.h"
+#include <gtest/gtest.h>
+
+using namespace Hedera;
+
+class NodeAddressBookTest : public ::testing::Test
+{
+protected:
+  void SetUp() override {}
+};
+
+TEST_F(NodeAddressBookTest, readAddressBookResource)
```

Comment:
What is this test actually testing? Should it check each address and make sure they're parsed correctly?

Reply:
Oops, this was just my debugging entry point. I'll remove the file for now, since I haven't written any meaningful tests